server/src/main/resources/c4.puml

```diff
@@ -730,7 +730,7 @@ $elementSkin
 ' Line breaks
 ' ##################################
 
-' PlantUML supports no DETERMINISTIC/automatic line breaks of "PlantUML line" (C4 Relashionships)
```

Comment:
File copied from: https://github.com/plantuml-stdlib/C4-PlantUML/blob/0d97fb8713bfafbc6bb231e036d4a74388353c23/C4.puml#L872

Reply:
> File copied from: https://github.com/plantuml-stdlib/C4-PlantUML/blob/0d97fb8713bfafbc6bb231e036d4a74388353c23/C4.puml#L872

Thanks for pointing that out. Meanwhile I authored this [pull request](https://github.com/plantuml-stdlib/C4-PlantUML/pull/292) that addresses the issue in the upstream repository. My PR was approved already 😄.